packages/ui/components/pages/Fuse/FusePoolsPage/FusePoolList/index.tsx

```diff
@@ -10,11 +20,41 @@ const FusePoolList = () => {
   const enabledChains = useEnabledChains();
   const { isLoading, poolsPerChain, error } = useCrossFusePools([...enabledChains]);
   const [err, setErr] = useState<Err | undefined>(error as Err);
+  const mounted = useRef(false);
+  const [preSearchText, setPreSearchText] = useState<string | undefined>(undefined);
+  const [preFilter, setPreFilter] = useState<(string | SupportedChains)[] | undefined>(undefined);
+  const [preSorting, setPreSorting] = useState<SortingState | undefined>(undefined);
+  const { address } = useMultiMidas();
 
   useEffect(() => {
     setErr(error as Err);
   }, [error]);
 
+  useEffect(() => {
+    mounted.current = true;
+
+    let _initSearch = localStorage.getItem(MIDAS_POOL_SEARCH);
+    _initSearch = _initSearch ? _initSearch : '';
+    const filterItem = localStorage.getItem(MIDAS_POOL_FILTER);
+    const _initFilter = filterItem
+      ? (JSON.parse(filterItem) as (string | SupportedChains)[])
+      : [ALL];
+    let _initSortId = localStorage.getItem(MIDAS_POOL_SORT_ID);
+    _initSortId = _initSortId ? _initSortId : address ? 'supplyBalance' : 'totalSupplied';
+    const _initSortDesc = localStorage.getItem(MIDAS_POOL_SORT_DESC);
+    const initSortDesc = _initSortDesc && _initSortDesc === 'true' ? true : false;
+
+    if (mounted.current) {
+      setPreSearchText(_initSearch);
+      setPreFilter(_initFilter);
+      setPreSorting([{ id: _initSortId, desc: initSortDesc }]);
+    }
+
+    return () => {
+      mounted.current = false;
+    };
+  }, [address]);
```

Comment:
Why execute this useEffect on "address" change? The config in the localstorage is not address specific. 

If you only want it to be run once it's mounted it can be just `[]`.

Reply:
Fixed